packages/vscode/src/providers.ts

```diff
@@ -70,7 +70,11 @@ export const createDefinitionsProvider = (
     doc: vscode.TextDocument,
     position: vscode.Position,
   ): Promise<vscode.Definition> => {
-    const currentToken = doc.getText(doc.getWordRangeAtPosition(position, /[\w.]+/))
+    let tokenPosition = doc.getWordRangeAtPosition(position, /"[^"]+"/)
+    if (_.isUndefined(tokenPosition)) {
+      tokenPosition = doc.getWordRangeAtPosition(position, /[\w.]+/)
+    }
```

Comment:
```suggestion
    const tokenPosition = doc.getWordRangeAtPosition(position, /"[^"]+"/) ?? 
      doc.getWordRangeAtPosition(position, /[\w.]+/)
```

Its just nicer without the let :)